parallel-install/pkg/deployment/coredns_test.go

```diff
@@ -5,50 +5,83 @@ import (
 
 	"github.com/kyma-incubator/hydroform/parallel-install/pkg/logger"
 	"github.com/stretchr/testify/require"
+	v1apps "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes/fake"
 )
 
 func Test_patchCoreDNS(t *testing.T) {
 	log := logger.NewLogger(true)
-	coreDNSCM := fakeCoreDNSCM()
+	coreDNSConfigMap := fakeCoreDNSConfigMap()
+	coreDNSDeployment := fakeCoreDNSDeployment()
 
-	t.Run("test skipping coreDNS patch", func(t *testing.T) {
+	t.Run("test skipping coreDNS patch when coreDNS deployment doesn't exist", func(t *testing.T) {
 		// given
-		kubeClient := fake.NewSimpleClientset(coreDNSCM)
+		kubeClient := fake.NewSimpleClientset(coreDNSConfigMap)
 
 		// when
-		cm, err := patchCoreDNS(kubeClient, `(.*)\.loacal\.kyma\.dev`, log)
+		cm, err := patchCoreDNS(kubeClient, `(.*)\.local\.kyma\.dev`, log)
```

Comment:
You repeat the same domain to almost every test case. Maybe it would be worth setting once in before-tests section?